### PR TITLE
Dev mock layout

### PR DESCRIPTION
**Pull request by raisarijasa**

### CODE REVIEW

File: app/src/main/res/layout/activity_intro.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                android:id="@+id/llIntroFirst"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:gravity="center"
+                android:orientation="vertical">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Fist Intro"
```
Comment (erdyevandy) [inline app/src/main/res/layout/activity_intro.xml:30]: A little bit typo

**Comment by erdyevandy** [comment]
Screenshot_2020-05-28-08-19-53-77_e8d761c83f86eb7b6512a5d623381fd0

I found this error when first opening the screen mas Rai, need to add action for user to turn on bluetooth when first launch

**Comment by raisarijasa** [comment]
@erdyevandy that should support device validation, does your device have Bluetooth LE feature?

**Comment by erdyevandy** [comment]
Yes mas @raisarijasa, because when I turn on bluetooth it works normally.
This problem occur only when our device bluetooth is turned off, because on the first screen app didn't suggest user to turn on bluetooth until user reach start service menu

**Comment by raisarijasa** [comment]
@erdyevandy the apps should not suggest the user permission there, only validation if the device supported Bluetooth or not.

**Pull request merged**